Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/settings/notifications/NotificationSettingsFragmentViewModel.kt

```diff
@@ -21,6 +30,14 @@ class NotificationSettingsFragmentViewModel @AssistedInject constructor(
         .map { NotificationSettingsState(it) }
         .asLiveData(dispatcherProvider.Default)
 
+    fun createNotificationChannels() {
+        if (BuildVersionWrap.hasAPILevel(Build.VERSION_CODES.O)) {
```

Comment:
We can get rid of these checks when we rely on `NotificationManagerCompat`, `presenceTracingNotifications, digitalCovidCertificateNotifications` are already using it ,but `generalNotifications`
Look at https://github.com/corona-warn-app/cwa-app-android/pull/4169